lib/src/components/text_component.dart

```diff
@@ -28,8 +29,14 @@ class TextComponent extends PositionComponent {
     _updateBox();
   }
 
-  TextComponent(this._text, {TextConfig config})
-      : _config = config ?? TextConfig() {
+  TextComponent(
+    this._text, {
+    TextConfig config,
+    Vector2 position,
+    Vector2 size,
+  })  : assert(_text != null),
+        _config = config ?? TextConfig(),
+        super(position: position) {
```

Comment:
`size` needs to be passed to super as well.

Reply:
fixed